wgengine/wgcfg/config_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package wgcfg
+
+import (
+	"reflect"
+	"testing"
+)
+
+// Tests that [Config.Equal] tests all fields of [Config], even ones
+// that might get added in the future.
+func TestConfigEqual(t *testing.T) {
```

Comment:
I started to write a comment asking for this test, and here it is. 😆

Reply:
GitHub code reviews need all our Slack emoji.